### PR TITLE
added new feature

### PR DESCRIPTION
**Pull request by andrii-kukuruza**
test message

### CODE REVIEW

**Comment by my-git-test-acc** [comment]
test-answer

File: test.txt [hunk 1]
```diff
@@ -1 +1 @@
-test
```
Comment (andrii-kukuruza) [inline test.txt:1]: hmm.. ?

**Review by andrii-kukuruza — COMMENTED** [review]
review-test-msg

File: test.txt (hunk 1 above)
Comment (my-git-test-acc) [reply test.txt:1]: ok

**Pull request merged**